introduction.md

```diff
@@ -0,0 +1,3 @@
+I am Blaze,
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
My favorite Code School paths are PHP, Git and Database. These are the paths I am on right now and I believe they are the foundations for what’s about to come.